make/00_mod.mk

```diff
@@ -43,12 +43,9 @@ api_docs_package := $(repo_name)/pkg/apis/trust/v1alpha1
 api_docs_branch := main
 
 helm_chart_source_dir := deploy/charts/csi-driver-spiffe
-helm_chart_name := cert-manager-csi-driver-spiffe
+helm_chart_image_name := quay.io/jetstack/charts/cert-manager-csi-driver-spiffe
 helm_chart_version := $(VERSION)
 helm_labels_template_name := cert-manager-csi-driver-spiffe.labels
-helm_docs_use_helm_tool := 1
-helm_generate_schema := 1
-helm_verify_values := 1
```

Comment:
question: Why are these removed? They don't seem related to what we're changing - are they defaulted on and therefore redundant to set here?

Reply:
Yes, this is cleanup, the variables have been removed here: https://github.com/cert-manager/makefile-modules/pull/218